dev-cpp/fbthrift/fbthrift-2024.11.04.00.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# These must be bumped together:
+# dev-cpp/edencommon
+# dev-cpp/fb303
+# dev-cpp/fbthrift
+# dev-cpp/fizz
+# dev-cpp/folly
+# dev-cpp/mvfst
+# dev-cpp/wangle
+# dev-util/watchman
+
+inherit cmake
+
+DESCRIPTION="Facebook's branch of Apache Thrift (C++ bindings)"
+HOMEPAGE="https://github.com/facebook/fbthrift"
+SRC_URI="https://github.com/facebook/fbthrift/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	app-arch/zstd:=
+	~dev-cpp/fizz-${PV}:=
+	~dev-cpp/folly-${PV}:=
+	dev-cpp/gflags:=
+	dev-cpp/glog:=[gflags]
+	~dev-cpp/mvfst-${PV}:=
+	~dev-cpp/wangle-${PV}:=
+	dev-libs/boost:=[nls(+)]
+	dev-libs/libevent:=
+	dev-libs/openssl:=
+	dev-libs/xxhash:=
+	sys-libs/zlib:=
+"
+RDEPEND="
+	${DEPEND}
+	test? ( dev-cpp/gtest )
+"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_DIR="$(get_libdir)/cmake/${PN}"
+		-DLIB_INSTALL_DIR="$(get_libdir)"
+		-Denable_tests=$(usex test 'ON' 'OFF')
+		-Wno-dev
+	)
+	cmake_src_configure
+}
+
+src_test() {
+	# See https://github.com/facebook/fbthrift/issues/628
```

Comment:
Please do `RESTRICT="test"` instead, with this comment, rather than a `src_test` which calls `die`.

Reply:
Should I also get rid of `USE="test"`? The expectation is that once upstream answers we'd re-enable the tests, and I'd rather just lift the restriction instead of adding the USE flag anew.

Reply:
Yeah, that should be fine.

Reply:
Alright, just replaced `RESTRICT="!test? ( test )"` with `RESTRICT="test"` for now.